agent/engine/containermetadata/metadata_manager_windows.go

```diff
@@ -0,0 +1,75 @@
+// +build windows
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package containermetadata
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/api"
+)
+
+const (
+	mountPoint = `C:\ecs\metadata`
```

Comment:
@cabbruzzese Is there something more standard for Windows?  I would assume something like `%ProgramData%\Amazon\ECS\metadata` but maybe that's not actually a good place?